src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -91,4 +85,12 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
       this.dtTrigger.next();
     }
   }
+
+  isLinkExternal(uri: string): WindowProxy | string {
+    return this.isValidUri(uri) ? window.open(uri, '_blank') : '';
+  }
+
+  isValidUri(uri: string) {
+    return uri.startsWith('http' || 'https');
```

Comment:
I think this could be `uri.startsWith('http://' || 'https://');` or use a regular expression to evaluate that